arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -62,6 +62,7 @@ void MaintenanceFeature::init() {
   requiresElevatedPrivileges(false); // ??? this mean admin priv?
 
   // these parameters might be updated by config and/or command line options
+
   _maintenanceThreadsMax = (std::max)(static_cast<uint32_t>(2),
```

Comment:
Please set this to 3, otherwise we might get https://github.com/arangodb/arangodb/pull/6310 again.